libft/sources/strings/strings.h

```diff
@@ -3,16 +3,18 @@
 /*                                                        :::      ::::::::   */
 /*   strings.h                                          :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/08/13 20:55:01 by inikulin          #+#    #+#             */
-/*   Updated: 2024/09/22 22:35:17 by inikulin         ###   ########.fr       */
+/*   Updated: 2024/10/02 11:36:13 by taretiuk         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #ifndef STRINGS_H
 # define STRINGS_H
 
+#include "../../sources/strings/strings_internal.h"
```

Comment:
I see. So the user of your function DOES need the structure, it's a parameter. In this case you might as well include the whole content of that _internal.h here. The point of _internal.h is to NOT expose it to the user. If _internal.h is included here, and this file is included in libft.h, this means that the libft user sees declarations from _internal.h. Which makes it not _internal_ :)